package_io/input_parser.star

```diff
@@ -96,6 +96,13 @@ def parse_input(input_args):
 	if result["network_params"]["deneb_fork_epoch"] == 0:
 		fail("deneb_fork_epoch is 0 needs to be > 0 ")
 
+	if result["network_params"]["electra_fork_epoch"] != None:
+		# if electra is defined, then deneb needs to be set very high
+		result["network_params"]["deneb_fork_epoch"] = 200000
```

Comment:
lets take out the constant 20000 and put it in a value up top like HIGH_DENEB_VALUE_FORK_VERKLE = 20000

Reply:
ok